solidity/random-beacon/contracts/Reimbursable.sol

```diff
@@ -28,9 +27,13 @@ abstract contract Reimbursable is Ownable {
         reimbursementPool.refund(gasStart - gasleft(), receiver);
     }
 
+    modifier onlyReimbursableAdmin() virtual {
+        _;
+    }
```

Comment:
I would hold on until #2860 is merged to see if `RandomBeacon` contract size allows for this kind of flexibility. If not, I would be also fine with having `Reimbursable` extend upgradeable `Ownable`.

Reply:
It looks like the contract size allows for this change. 
`main` branch is currently at `24.394`. I did a little bit of testing in RandomBeacon and after the changes, it would be around `24.220`

I think we should open a draft PR (which we'll need to do anyway if this PR lands in `main`) that branches off of this branch and add the `onlyReimbursableAdmin` modifier and probably `Governable.sol` and see how it impacts the RandomBeacon contract size. Something similar to what we have in https://github.com/keep-network/keep-core/pull/2921 then we can decide if we should merge the changes proposed here or not.

Reply:
Done in https://github.com/keep-network/keep-core/pull/2947